src/main/java/com/direwolf20/buildinggadgets/common/tools/GadgetUtils.java

```diff
@@ -300,9 +300,9 @@ public static boolean anchorBlocks(EntityPlayer player, ItemStack stack) {
             }
             List<BlockPos> coords = new ArrayList<BlockPos>();
             if (stack.getItem() instanceof GadgetBuilding) {
-                coords = BuildingModes.getBuildOrders(world, player, startBlock, sideHit, stack); //Build the positions list based on tool mode and range
+                coords = BuildingModes.getBuildCoords(world, player, startBlock, sideHit, stack); //Build the positions list based on tool mode and range
```

Comment:
Please can we use fully formed words in important methods or better yet 
```suggestion
                coords = BuildingModes.getBlockPosList(world, player, startBlock, sideHit, stack); //Build the positions list based on tool mode and range
```

or something like that, I've not taken a long look at the actual method but something that some what describes the method is preferred ( yes this isn't currently a start but we've got to start somewhere 😆 )

Reply:
Just checked, you're literally handing back a List of Block Pos's haha. so yeah lets keep it descriptive: List<BlockPos> getBlockPosList

Reply:
How about `getBuildPositions`? I don't like suffix/prefix method with their return type

Reply:
getBlockPositions ? because we're not returning a build position we're retting a list of literal block pos :P

Reply:
But they are used for positions to place block.
Or we'll see... I think this method will be gone in the near future

Reply:
You don't really write method names with that thinking. You describe what they do, not what something else is using them for.